lib/utils/keymaps-normalize.js

```diff
@@ -0,0 +1,23 @@
+function normalize(keybinding) {
+  function sortAlphabetically(a, b) {
+    return a.localeCompare(b);
+  }
+
+  return keybinding.split('+').sort(sortAlphabetically).join('+').toLowerCase();
+}
+
+function getNormalizedKeyMaps(userKeymaps) {
+  const normalized = {};
+
+  for (const keymap in userKeymaps) {
+    if (userKeymaps[keymap]) {
+      normalized[normalize(keymap)] = userKeymaps[keymap].toLowerCase();
+    }
+  }
+
+  return normalized;
+}
+
+module.exports.returnCommand = (keys, commands) => {
+  return getNormalizedKeyMaps(commands)[normalize(keys)];
```

Comment:
Maybe better to normalize keyMaps once and for all here: https://github.com/wilsontayar/hyper/blob/73d4d9e761a4a2823c1dd883c0ef96f1a4e0c8f4/app/config/keymaps.js#L10
`toLowerCase()` is already a kind of normalization.

Reply:
Definitely better to normalize once. I'll look into it. :)